src/otx/core/model/entity/classification.py

```diff
@@ -485,6 +584,8 @@ def __init__(
         use_throughput_mode: bool = True,
         model_api_configuration: dict[str, Any] | None = None,
     ) -> None:
+        model_api_configuration = model_api_configuration if model_api_configuration else {}
```

Comment:
Could you please make the same update for OVHlabelClassificationModel? I forgot that we create model in constructor